packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -0,0 +1,509 @@
+        <Table.HeaderCheckboxCell />
+        {TABLE_HEADERS.filter((head) => head.name !== 'name' || shouldDisplayMainField).map(
+          (head) => (
+            <Table.HeaderCell key={head.name} {...head} />
+          )
+        )}
+      </Table.Head>
+      <Table.Loading />
+      <Table.Body>
+        {rowsToDisplay.map((row, index) => (
+          <Table.Row key={row.id}>
+            <Table.CheckboxCell id={row.id} />
+            <Table.Cell>
+              <Typography>{row.id}</Typography>
+            </Table.Cell>
+            {shouldDisplayMainField && (
+              <Table.Cell>
+                <Typography>{row[mainField as keyof TableRow]}</Typography>
+              </Table.Cell>
+            )}
+            <Table.Cell>
+              <DocumentStatus status={row.status} maxWidth={'min-content'} />
+            </Table.Cell>
+            <Table.Cell>
+              {isPublishing && entriesToPublish.includes(row.documentId) ? (
+                <Flex gap={2}>
+                  <Typography>
+                    {formatMessage({
+                      id: 'content-manager.success.record.publishing',
+                      defaultMessage: 'Publishing...',
+                    })}
+                  </Typography>
+                  <Loader small />
+                </Flex>
+              ) : (
+                <EntryValidationText
+                  validationErrors={validationErrors[row.documentId]}
+                  isPublished={row.publishedAt !== null}
+                />
+              )}
+            </Table.Cell>
+            <Table.Cell>
+              <IconButton
+                tag={Link}
+                to={{
+                  pathname: `${pathname}/${row.documentId}`,
+                  search: row.locale && `?plugins[i18n][locale]=${row.locale}`,
+                }}
+                state={{ from: pathname }}
+                label={formatMessage(
+                  { id: 'app.component.HelperPluginTable.edit', defaultMessage: 'Edit {target}' },
+                  {
+                    target: formatMessage(
+                      {
+                        id: 'content-manager.components.ListViewHelperPluginTable.row-line',
+                        defaultMessage: 'item line {number}',
+                      },
+                      { number: index + 1 }
+                    ),
+                  }
+                )}
+                target="_blank"
+                marginLeft="auto"
+              >
+                <Pencil />
+              </IconButton>
+            </Table.Cell>
+          </Table.Row>
+        ))}
+      </Table.Body>
+    </Table.Content>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * BoldChunk
+ * -----------------------------------------------------------------------------------------------*/
+
+const BoldChunk = (chunks: React.ReactNode) => <Typography fontWeight="bold">{chunks}</Typography>;
```

Comment:
at the end the chunks prop is the children prop, isn't it?

Reply:
Yes, but I think formatMessage pass chunks as the children